argcomplete/_check_module.py

```diff
@@ -12,25 +12,32 @@
 
     def find_spec(name):
         """Minimal implementation as required by `find`."""
-        f, path, _ = find_module(name)
+        try:
+            f, path, _ = find_module(name)
+        except ImportError:
+            return None
         has_location = path is not None
         if f is None:
             return ModuleSpec(None, has_location, [path])
         f.close()
         return ModuleSpec(path, has_location, None)
 
 
+class LocateFailure(RuntimeError):
```

Comment:
Better name suggestions appreciated.

Reply:
Perhaps `CheckFailure` or `MarkerNotFound` given that it's a generic exception to indicate that *something* has gone wrong during the process of checking the given module name to see if it contains the PYTHON_ARGCOMPLETE_OK marker. Don't think I have a strong preference for any of these three though.

Reply:
I like `MarkerNotFound`. I also thought about  `CannotArgcomplete` or even `NoArgcomplete`.

Reply:
They're all fine by me. I'm happy for @kislyuk to weigh in with a preference, or you can take your pick.

Reply:
I would go with `ArgcompleteMarkerNotFound`.

Reply:
Amended.